### DISCUSSION TITLE
robots metadata per node

### DISCUSSION THREAD

**Discussion by eliranrefael** [patrickdemooij9/SeoToolkit.Umbraco#174] [Q&A]
hey,
i implemented my own properties to generate robots metadata for each node with index and follow properties.
the problem is that the tool adds its own robots empty metadata in tags and im afraid that two tags will be wrong for the site functionality.
any idea how to solve it?

thanks

**Comment by patrickdemooij9** [comment]
Hi @eliranrefael 
I am not 100% sure, but I think you should be able to do the following to turn the robots fields off:
```
public class TestComposer : IComposer
    {
        public void Compose(IUmbracoBuilder builder)
        {
            builder.WithCollectionBuilder<SeoFieldCollectionBuilder>()
                .Remove<RobotsField>();
        }
    }
```

Let me know if this works for you, otherwise I'll see what changes are required to make this work for you.

**Reply by eliranrefael** [reply]
it works! great!
